client/gutenberg/extensions/map/edit.js

```diff
@@ -196,7 +204,7 @@ class MapEdit extends Component {
 			</Placeholder>
 		);
 		const getAPIInstructions = sprintf(
-			"<p>Before you use a map block, you will need to get a key from <a href='%1$s'>Mapbox</a>. You will only have to do this once.</p><p>Go to <a href='%1$s'>Mapbox</a> and either create an account or sign in. Once you sign in, locate and copy the default access token. Finally, paste it into the token field below.</p>",
+			"<p>Before using the map block, you will need an Access Token.</p><p>Create an account or log in to <a href='%1$s'>Mapbox</a>. Locate and copy the default access token, and paste it in the field below.</p>",
```

Comment:
Shipping non-localized strings is a problem.

Reply:
Would this work instead?
```jsx
const getAPIInstructions = (
  <Fragment>
    <p>{ __( 'Before using the map block, you will need an Access Token.' ) }</p>
    <p>
      { sprintf(
        __( 'Create an account or log in to %s. Locate and copy the default access token, and paste it in the field below.' ),
        '<a href="https://www.mapbox.com">Mapbox</a>'
      ) }
    </p>
  </Fragment>
);
```

Reply:
Ah sorry, that probably won't work since `__()` cannot take HTML, so maybe:

Would this work instead?
```jsx
const getAPIInstructions = (
  <Fragment>
    <p>{ __( 'Before using the map block, you will need an Access Token.' ) }</p>
    <p>
      <a href="https://www.mapbox.com">
        { __( 'Create an account or log in to Mapbox.' ) }
      </a>
      { __( 'Locate and copy the default access token, and paste it in the field below.' ) }
    </p>
  </Fragment>
);
```

...until Gutenberg gets HTML in translations support?

Reply:
Would be also neat to use [`ExternalLink`](https://github.com/WordPress/gutenberg/blob/5b81cbc66c5d52f9603f77a60ec7864150173dd0/packages/components/src/external-link/README.md) component for the link since opening in the same page is kinda jarring as the user is in a middle of editing the page.

We also get "external link" icon which is consistent with rest of the Gutenberg.

So at the end, I would replace this:

https://github.com/Automattic/wp-calypso/blob/5624e2ab96704fbed5830a3e8155690c0cf934c4/client/gutenberg/extensions/map/edit.js#L213-L215

With this:
```jsx
<div className="components-placeholder__instructions">
  <p>{ __( 'Before using the map block, you will need an Access Token.' ) }</p>
  <p>
    <ExternalLink href="https://www.mapbox.com">
      { __( 'Create an account or log in to Mapbox.' ) }
    </ExternalLink>
    { __( 'Locate and copy the default access token, and paste it in the field below.' ) }
  </p>
</div>
```

Resulting in this:

<img width="909" alt="screenshot 2018-11-20 at 23 29 27" src="https://user-images.githubusercontent.com/87168/48804104-6a1db880-ed1c-11e8-9b84-840de0029c29.png">

_(I have TwentyNineteen theme active and hence so big text size)_

Reply:
cc @mapk and @MichaelArestad since that turns this into kinda design review ☝️

Reply:
That looks okay for the V1. Can you tweak the copy at the same time to be:

> Locate and copy the "default access token."
> Then, paste it into the field below.

Reply:
Done in: https://github.com/Automattic/wp-calypso/pull/28730/commits/561721c85425c5648c57cb158a169b2f5008cd31